added additional context for ExpressRoute UDR (#90)
microsoft/azure-docs: articles/logic-apps/connect-virtual-network-vnet-isolated-environment.md

  * Make sure that your virtual network [enables access for your ISE](#enable-access) so that your ISE can work correctly and stay accessible.

>   * If you use [ExpressRoute](../expressroute/expressroute-introduction.md), which provides a private connection to Microsoft cloud services that's facilitated by the connectivity provider, you must make sure that you are not using forced tunnel configuration. If this is enabled, you must [create a route table](../virtual-network/manage-route-table.md) that has the following route and link that table to each subnet that's used by your ISE:

Reviewer: Thanks, @nthewara! The team has approved this line item update, so here are the edits to make:

```suggestion
  * If you use [ExpressRoute](../expressroute/expressroute-introduction.md), which provides a private connection to Microsoft cloud services that's facilitated by the connectivity provider, make sure that you're not using [forced tunneling](../app-service/app-service-security-controls.md). Otherwise, if this capability is enabled, you must [create a route table](../virtual-network/manage-route-table.md) that has the following route and then link the route table to each subnet that's used by your ISE:
```

>   * If you use [ExpressRoute](../expressroute/expressroute-introduction.md), which provides a private connection to Microsoft cloud services that's facilitated by the connectivity provider, make sure that you're not using [forced tunneling](../app-service/app-service-security-controls.md). Otherwise, if this capability is enabled, you must [create a route table](../virtual-network/manage-route-table.md) that has the following route and then link the route table to each subnet that's used by your ISE:

Reviewer: Just wanted to replace "forced tunneling" with a better link:

```suggestion
  * If you use [ExpressRoute](../expressroute/expressroute-introduction.md), which provides a private connection to Microsoft cloud services that's facilitated by the connectivity provider, make sure that you're not using [forced tunneling](../firewall/forced-tunneling.md). Otherwise, if this capability is enabled, you must [create a route table](../virtual-network/manage-route-table.md) that has the following route and then link the route table to each subnet that's used by your ISE:
```

Reviewer: Hi @nthewara, you missed this edit.

    **Name**: <*route-name*><br>
    **Address prefix**: 0.0.0.0/0<br>

>     **Next hop**: Internet

Reviewer: Please add this link below **Next hop**, including the blank lines. Thanks!

```suggestion
    **Next hop**: Internet
    
    For more information about this route, see [0.0.0.0/0 address prefix](../virtual-network/virtual-networks-udr-overview.md#default-route).
     
```

Reviewer: Here's another missed edit.

>    * If you use Network Virtual Appliances, make sure that you have not enabled SSL termination or traffic inspection for traffic originating from ISE Subnet.

Reviewer: @nthewara: Review still pending.

Reviewer: I meant, the line about network virtual appliances is still pending review by the engineering team.

Reviewer: This comment actually isn't resolve yet.

>    * If you use Network Virtual Appliances, make sure that you have not enabled SSL termination or traffic inspection for traffic originating from ISE Subnet.

Reviewer: Adding feedback from team and style edits:

```suggestion
   * If you use a [network virtual appliance (NVA)](../virtual-network/virtual-networks-udr-overview.md#user-defined), make sure that you don't enable TLS/SSL termination or change the outbound TLS/SSL traffic. Also, make sure that you don't enable inspection for traffic that originates from your ISE's subnet. For more information, see [Virtual network traffic routing](../virtual-network/virtual-networks-udr-overview.md).
```
